bugbug/utils.py

```diff
@@ -558,3 +558,54 @@ def escape_markdown(text: str) -> str:
 def keep_as_is(x):
     """A tokenizer that does nothing."""
     return x
+
+
+def get_keyword_dict() -> dict:
+    """Returns a dictionary mapping keywords to bug types."""
+    return {
+        "sec-": "security",
+        "csectype-": "security",
+        "memory-": "memory",
+        "crash": "crash",
+        "crashreportid": "crash",
+        "perf": "performance",
+        "topperf": "performance",
+        "main-thread-io": "performance",
+        "power": "power",
+    }
```

Comment:
Modifying this PR should be fine since we are still within the scope of the issue. At that point, `bug_to_types` itself should be moved as well. Since we are on it, I would rename the function to something more descriptive like `identify_bug_types` or `infer_bug_types`.